Performance/QPSBenchmark/Package.swift

```diff
@@ -53,14 +53,9 @@ let package = Package(
         .product(name: "SwiftProtobuf", package: "swift-protobuf"),
         .target(name: "BenchmarkUtils"),
       ],
-      exclude: [
-        "Model/benchmark_service.proto",
-        "Model/control.proto",
-        "Model/core_stats.proto",
-        "Model/messages.proto",
-        "Model/payloads.proto",
-        "Model/stats.proto",
-        "Model/worker_service.proto",
```

Comment:
Previously the excludes were required to stop SPM warning about unused source files; does it no longer warn? (I have a feeling it only did it on clean builds but I could totally be wrong)

Reply:
I tried cleaning and building several times and I got no warnings, that's why I decided to get rid of them.

Reply:
These files are not unused anymore since they are input files to the plugin invocation.